<launch>
	<group ns="$(arg veh)">
		<node pkg="led_emitter" type="led_emitter_node.py" name="led_emitter_node" output="screen">
			<remap from="~change_light_frequency" to="led_emitter_node_test/change_light_frequency"/>
		</node>
		<node pkg="led_emitter" type="led_emitter_node_test.py" name="led_emitter_node_test" output="screen">
			<remap from="~current_led_state" to="led_emitter_node/current_led_state"/>
		</node>
	</group>
</launch>
